README.md

```diff
@@ -32,4 +32,9 @@ support will not be provided. Still configuring rocket to use infoblox cni plugi
 Development
 -----------
 
-* Build - For dependencies and build instructions, refer to [BUILD.md](docs/BUILD.md) .
\ No newline at end of file
+* Build - For dependencies and build instructions, refer to [BUILD.md](docs/BUILD.md) .
+
+Use Existing Network
+--------------------
```

Comment:
1) Tell what is happening now.
2) Tell why and which EA's we have to update.
3) Give examples see my comment in JIRA